docs/scripts/devel-dependency-containers/docker-compose-mariadb.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+services:
+  elasticsearch:
+    container_name: opencast-elasticsearch
+    image: docker.io/library/elasticsearch:7.5.1
+    ports:
+      - 127.0.0.1:9200:9200
+      - 127.0.0.1:9300:9300
+    environment:
+      discovery.type: single-node
+  activemq:
+    container_name: opencast-activemq
+    image: quay.io/lkiesow/activemq
```

Comment:
Do you actively maintain that? I'm not really satisfied with the current ActiveMQ images I use and I wanted to have an "Opencast ready" version for some time now. Would be great if that can also be used in production.

Reply:
By discussion in chat: Long term we want to push a well supported ActiveMQ container to Opencast's quay.io account